src/user/user.controller.ts

```
@@ -1,11 +1,29 @@
+
   @Post()
   async create(@Body() user: User): Promise<User> {
-    return { ...user, id: 1000 };
+    return this.repository.create(user);
+  }
+
+  @Get(':id')
+  async findOne(@Param('id') id: string): Promise<User> {
+    const userID = Number.parseInt(id);
+
+    if (isNaN(userID)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)